config/stack/manifests/resources/cockroachclusterclass.ui-schema.yaml

```diff
@@ -0,0 +1,5 @@
+version: 0.4
+configSections: []
+
+resourceConditionPaths:
+- .status.conditionedStatus.conditions[?(@.type=='Ready')]
```

Comment:
same comment as in the stack-gcp PR, I don't think classes have any status or conditions. this applies to the yugabyte class too.